e2-studio-star-rx72n-firmware/tests/test_rx_frame_ascii.c

```
@@ -758,7 +831,7 @@ void test_type_name_unknown(void)
 void test_type_name_invalid(void)
 {
   /* Invalid type value should return UNKNOWN */
-  const char* name = rx_frame_ascii_type_name((rx_frame_type_t)0xFF);
+  const char* name = rx_frame_ascii_type_name((rx_frame_type_t)0x50);
```

Clang (14.0.6)

[warning] variable 'name' is not initialized

(cppcoreguidelines-init-variables)